src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
I'm a bit hesitant about using `info args` for debuginfo tests. A `gdb-check` instruction just verifies that the argument can be found *somewhere* in the output, so without the numbering it's possible to overlook errors. For example, given the following program...
```rust
let x = 1;
breakpoint();
{
    let x = 2;
    breakpoint();
}
breakpoint();
```
... we get a correct result with `print`:
```
gdb-command:print x
gdb-command:continue
gdb-command:print x
gdb-command:continue
gdb-command:print x
gdb-check:$1 = 1
gdb-check:$2 = 2
gdb-check:$3 = 2

produces:
$1 = 1
$2 = 2
$3 = 2 // let's say there's a bug in scoping and x still refers to wrong address
```
The error will be detected because there is not `$3 = 1` in the output. But with `info`, the overlapping names might hide the error:
```
gdb-command:info locals
gdb-command:continue
gdb-command:info locals
gdb-command:continue
gdb-command:info locals
gdb-check:x = 1
gdb-check:x = 2
gdb-check:x = 1 // <-- this is actually redundant

produces:
x = 1
x = 2
x = 2 // the test would still pass, because `x = 1` can be found in the output.
```